app.js

```diff
@@ -54,7 +54,7 @@ app.use((err, req, res, next) => {
   if (err.msg && err.status) {
     res.status(err.status).send({ msg: err.msg });
   }
-  // console.log(err);
+  console.log(err);
   next(err);
 });
 
```

Comment:
Looks good! Highly recommend that you extract all your error handlers and put them all in a separate JS file so your app.js is cleaner 😄